File: trunk/CSClient/Library/Library.Model/Entriy/Audit.cs
/**  版本信息模板在安装目录下，可自行修改。
* Audit.cs
*
* 功 能： N/A
* 类 名： Audit
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2015/7/1 2:29:06   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace Library.Model
{
	/// <summary>
	/// Audit:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class Audit
	{
		public Audit()
		{}
		#region Model
		private int _f_id;
		private string _f_name;
		private string _f_content;
		private decimal _f_money;
		private DateTime _f_time;
		private string _f_wf_stateid;
		private int _f_userid;
		private bool _f_isdelete= false;
		private DateTime _f_date;
		private int _f_type=0;
		/// <summary>
		/// 
		/// </summary>
		public int F_ID
		{
			set{ _f_id=value;}
			get{return _f_id;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_Name
		{
			set{ _f_name=value;}
			get{return _f_name;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_Content
		{
			set{ _f_content=value;}
			get{return _f_content;}
		}
		/// <summary>
		/// 
		/// </summary>
		public decimal F_Money
		{
			set{ _f_money=value;}
			get{return _f_money;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime F_Time
		{
			set{ _f_time=value;}
			get{return _f_time;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_WF_StateID
		{
			set{ _f_wf_stateid=value;}
			get{return _f_wf_stateid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_UserID
		{
			set{ _f_userid=value;}
			get{return _f_userid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public bool F_IsDelete
		{
			set{ _f_isdelete=value;}
			get{return _f_isdelete;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime F_Date
		{
			set{ _f_date=value;}
			get{return _f_date;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_Type
		{
			set{ _f_type=value;}
			get{return _f_type;}
		}
		#endregion Model

	}
}

File: trunk/yyandroid/src/com/yy/android/vo/Audit.java
package com.yy.android.vo;

import java.io.Serializable;
import java.util.Date;

public class Audit implements Serializable {
	private static final long serialVersionUID = 8446360717274878127L;
	private int F_ID;
	private String F_Name;
	private String F_Content;
	private float F_Money;
	private Date F_Time = new Date();
	private String F_WF_StateID;
	private int F_UserID;
	private boolean F_IsDelete;
	private Date F_Date;
	private String Operator;
	private int F_Type;

	public int getF_Type() {
		return F_Type;
	}

	public void setF_Type(int f_Type) {
		F_Type = f_Type;
	}

	public String getOperator() {
		return Operator;
	}

	public void setOperator(String operator) {
		Operator = operator;
	}

	public int getF_ID() {
		return F_ID;
	}

	public void setF_ID(int in_F_ID) {
		F_ID = in_F_ID;
	}

	public String getF_Name() {
		return F_Name;
	}

	public void setF_Name(String in_F_Name) {
		F_Name = in_F_Name;
	}

	public String getF_Content() {
		return F_Content;
	}

	public void setF_Content(String in_F_Content) {
		F_Content = in_F_Content;
	}

	public float getF_Money() {
		return F_Money;
	}

	public void setF_Money(float in_F_Money) {
		F_Money = in_F_Money;
	}

	public Date getF_Time() {
		return F_Time;
	}

	public void setF_Time(Date in_F_Time) {
		F_Time = in_F_Time;
	}

	public String getF_WF_StateID() {
		return F_WF_StateID;
	}

	public void setF_WF_StateID(String in_F_WF_StateID) {
		F_WF_StateID = in_F_WF_StateID;
	}

	public int getF_UserID() {
		return F_UserID;
	}

	public void setF_UserID(int in_F_UserID) {
		F_UserID = in_F_UserID;
	}

	public boolean isF_IsDelete() {
		return F_IsDelete;
	}

	public void setF_IsDelete(boolean in_F_IsDelete) {
		F_IsDelete = in_F_IsDelete;
	}

	public Date getF_Date() {
		return F_Date;
	}

	public void setF_Date(Date in_F_Date) {
		F_Date = in_F_Date;
	}

}